src/components/ThemeProvider/README.md

```diff
@@ -0,0 +1,232 @@
+---
+name: Theme provider
+category: Structure
+keywords:
+  - theme
+  - provider
+  - color
+  - logo
+  - color scheme
+  - light mode
+  - dark mode
+omitAppProvider: true
```

Comment:
This is the secret sauce for turning off the wrapping `AppProvider` in Storybook and the Style Guide